docs/gno-docs/docs/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+id: overview
+slug: /
+---
+
+# Overview
+
+## What is Gno.land?
+
+Gno.land is a Layer 1 blockchain platform that enables the execution of Smart Contracts using an interpreted
+version of the Go programming language called Gnolang (Gno for short).
+
+### Key Features and Technology
+
+1. **Interpreted Gnolang**: Gno.land utilizes the Gnolang programming language, which is based on Go. It is executed
+   through a specialized virtual machine called the GnoVM, purpose-built for the Gno.land blockchain. While Gnolang
```

Comment:
> purpose-built for the Gno.land blockchain.
^ I was under the impression that others could run GnoVM and not have it be related to Gno.land, so perhaps keeping the language more generalized so that others know they can run a blockchain using GnoVM without it needing to be specifically tied to Gno.land. 

If the above is true, consider something along the lines of: "purpose-built for blockchain development with built-in determinism and a modified standard library to remove unnecessary packages like networking."

Reply:
That is correct, gnolang is chain agnostic.  

GNO is an on-chain interpreter language that stores and merkelizes deterministic values. Its human-readable Abstract Syntax Tree (AST) source code allows on-chain visualization and verification of the executed business logic by the Virtual Machine (VM), minimizing dependence on centralized third-party verification between source and byte codes.

Reply:
Rephrased:

[6d3e57c](https://github.com/gnolang/gno/pull/1046/commits/6d3e57c82e8ea6026f4d197712ba7164416d968f)